### PR TITLE
More trust updates

### PR DESCRIPTION
**Pull request by Maxjuhh123**

### CODE REVIEW

File: agents1/OfficialAgent.py [hunk 1]
```diff
@@ -686,7 +694,7 @@ def decide_on_actions(self, state):
                         self._goalVic in self._foundVictims and self._goalVic in self._todo and len(self._searchedRooms)==0 and 'class_inheritance' in info and 'CollectableBlock' in info['class_inheritance'] and 'mild' in info['obj_id'] and info['location'] in self._roomtiles:
                         objects.append(info)
                         # Remain idle when the human has not arrived at the location
-                        ## TODO (look through the code more first!): if you wait for a very long time, the human basically is lying
+                        ## TODO Max (look through the code more first!): if you wait for a very long time, the human basically is lying
```
Comment (BendikC) [inline agents1/OfficialAgent.py:703]: The decrementing part is a max job but I will do the check, working on it rn

File: beliefs/TrustBelief.py [hunk 2]
```diff
@@ -79,12 +79,17 @@ def should_trust(self, min_comp, min_will):
...
-        formula = self.competence*random_value + self.willingness*random_value
+        # TODO find proper weights based on literature
+        willingness_weight = 0.4
+        competence_weight = 0.6
+        competence_willingness_sum = self.competence*competence_weight + self.willingness*willingness_weight
 
-        return formula > 0.5
+        random_threshold = random.uniform(0, 1)
+
+        return competence_willingness_sum > random_threshold
```
Comment (BendikC) [inline beliefs/TrustBelief.py]: >= for the rare case that we end up with 1 in our random threshold (always trust wont trust in this case)

File: agents1/OfficialAgent.py [hunk 3]
```diff
@@ -632,6 +641,7 @@ def decide_on_actions(self, state):
                     self._todo.append(self._recentVic)
                     self._recentVic = None
                     self._phase = Phase.FIND_NEXT_GOAL
+                    trustBelief.decrement_willingness(0.02)
```
Comment (BendikC) [inline agents1/OfficialAgent.py:650]: Please add comments for why we are decrementing or incrementing, would help everyone

**Review by BendikC — CHANGES_REQUESTED** [review]

File: beliefs/TrustBelief.py (hunk 2 above)
Comment (Maxjuhh123) [reply beliefs/TrustBelief.py]: ooh yeah missed that case thanks

File: agents1/OfficialAgent.py (hunk 1 above)
Comment (Maxjuhh123) [reply agents1/OfficialAgent.py:703]: Let me know when it's done

**Pull request merged**